schemaregistry/schemaregistry_client.go

```diff
@@ -260,18 +264,34 @@ func NewClient(conf *Config) (Client, error) {
 		if err != nil {
 			return nil, err
 		}
+		latestToSchemaCache, err = cache.NewLRUCache(conf.CacheCapacity)
+		if err != nil {
+			return nil, err
+		}
 	} else {
 		schemaToIDCache = cache.NewMapCache()
 		idToSchemaCache = cache.NewMapCache()
 		schemaToVersionCache = cache.NewMapCache()
 		versionToSchemaCache = cache.NewMapCache()
+		latestToSchemaCache = cache.NewMapCache()
+	}
+	if conf.CacheLatestTTLSecs > 0 {
```

Comment:
I'm not familiar with the use case here, but we can have a LruCache with TTL on individual entries, why do we want to clear the entire cache vs having TTLs on each cache entry

Reply:
The TTL is just to ensure that the latest entries are refreshed every so often.  It's not necessary to have a TTL per entry.